src/ViewLayer/RaceModeDisplay/RaceModeDashboardView.h

```diff
@@ -8,6 +8,7 @@
 #include "../../DataLayer/MotorFaultsData/ErrorFlags.h"
 #include "../../DataLayer/MotorFaultsData/LimitFlags.h"
 #include "../../PresenterLayer/BatteryFaultsPresenter/BatteryFaultsPresenter.h"
+#include "../../PresenterLayer/KeyMotorPresenter/KeyMotorPresenter.h"
```

Comment:
You were probably just following what was happening before, but we're forward declaring keyMotorPresenter (line 21), this (and BatteryFaultsPresenter's) include lines can be moved to `RaceModeDashboardView.cpp`